RecoLocalCalo/HcalRecAlgos/interface/HLTv2.h

```diff
@@ -0,0 +1,63 @@
+#ifndef HLTv2_h
+#define HLTv2_h 1
+
+#include <typeinfo>
+#include <vector>
+
+// #include "DataFormats/HcalDetId/interface/HcalDetId.h"
+//#include "HcalPulseShapes.h"
+#include "CalibCalorimetry/HcalAlgos/interface/HcalTimeSlew.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/PedestalSub.h"
+
+#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+#include "CalibCalorimetry/HcalAlgos/interface/HcalPulseShapes.h"
+#include "CalibFormats/HcalObjects/interface/HcalCoder.h"
+#include "CalibFormats/HcalObjects/interface/HcalCalibrations.h"
+
+//#include <TMinuit.h>
+
+//#include <TH1.h>
+//#include "Minuit2/FCNBase.h"
+//#include "Minuit2/Minuit2Minimizer.h"
+#include "Math/Functor.h"
+#include "TF1.h"
+#include "TF2.h"
+#include "TMath.h"
```

Comment:
None of these 4 headers appear necessary for this file.
If they are needed in .cc, include them there.
It looks like TF1 and TF2 are not needed at all.